README.md

```diff
@@ -371,15 +373,24 @@ The directory we've mounted above will be accessible from the `/mnt/shared/proje
 
   To change the default network to `192.168.77.1`:
 
-  ```
+  ```shell
   sudo defaults write /Library/Preferences/SystemConfiguration/com.apple.vmnet.plist Shared_Net_Address -string 192.168.77.1
   ```
 
   Note that even through a network would normally be specified as `192.168.77.0`, the [vmnet framework](https://developer.apple.com/documentation/vmnet) seems to treat this as a starting address too and refuses to pick up such network-like values.
 
   The default subnet mask `255.255.255.0` should suffice for most use-cases, however, you can also change it to `255.255.0.0`, for example:
 
-  ```
+  ```shell
   sudo defaults write /Library/Preferences/SystemConfiguration/com.apple.vmnet.plist Shared_Net_Mask -string 255.255.0.0
   ```
 </details>
+
+<details>
+  <summary>How to connect to a VM over SSH?</summary>
+
+  If the guest VM is running and configured to accept incoming SSH connections you can conveniently connect to it like so:
+  ```shell
+  ssh admin:admin@$(tart ip monterey-base)
```

Comment:
I don't think it works like that: `ssh` will treat whole `admin:admin` as username.

Reply:
You're correct. I don't how I managed that 🤦 I use sshpass to do this personally. I'll fix it up.